iohk-monitoring/src/Cardano/BM/Backend/Log.lhs

```diff
@@ -407,10 +409,15 @@ mkTextFileScribe rotParams fdesc =
   where
     formatter :: (K.LogItem a) => Handle -> Rendering a -> IO Int
     formatter hdl r =
-        case KC._itemMessage (logitem r) of
-                K.LogStr ""  ->
-                    -- if message is empty do not output it
-                    return 0
+        let li = logitem r
+        in
+        case KC._itemMessage li of
+                K.LogStr ""  -> do
```

Comment:
This might be a type in the future?

Reply:
katip's message type contains _itemMessage (a textual message) and _itemPayload (in this case a JSON structure). When we output to textual logs we want to prioritise the text message and only fall back to the structure message if there is no textual representation.